cmp/grid/columns/Column.js

```diff
@@ -475,6 +475,8 @@ export class Column {
 
                 refresh() {return false}
             });
+        } else {
+            setRenderer((agParams) => agParams.value);
```

Comment:
Could we please add a comment on this line as to why we are installing a no-op renderer? I can easily imagine us forgetting that there is a difference in the generated markup between this passthrough renderer and no renderer at all (or remembering that there is _some_ reason why we have this but no longer being sure why, or if it applies in some future ag-grid version).